### PR TITLE
less agresive redaction

### PR DESCRIPTION
**Pull request by 10maurycy10**
[REDACTED]

### CODE REVIEW

**Comment by 10maurycy10** [comment]
identifyers from example.ts should be redacted.

File: README.md [hunk 1]
```diff
@@ -4,27 +4,27 @@
 Introduction: Sase is the world's *best* casing system. It's very early and experimental but it will catch on fast and will be used throughout the world as the universal casing system.  
 
 Case assumptions: 
-``_`` assumes private (not to be used publicly often)
```
Comment (entropylost) [inline README.md:7]: Redact the _ as well?

File: README.md [hunk 2]
```diff
@@ -4,27 +4,27 @@
...
 
 Expections:
-``_Capital_Capital`` can be made in languages that are more snakecase based like Rust and Python 
-``_Capital_Lowercase`` can also be accepted based on the situation but generally discouraged
+``[REDACTED]`` can be made in languages that are more snakecase based like Rust and Python 
+``[REDACTED]`` can also be accepted based on the situation but generally discouraged
 
 
-Note: ``Capital_Capital`` or``Capital_lowercase``  is HIGHLY discouraged but can be accepted
+Note: [REDACTED] or [REDACTED]  is HIGHLY discouraged but can be accepted
```
Comment (entropylost) [inline README.md:15]: mix in some [DATA EXPUNGED] and ███s to make it less boring

**Review by entropylost — APPROVED** [review]
Mostly okay.

**Comment by entropylost** [comment]
@ZeroTixDev Please merge.

**Comment by ZeroTixDev** [comment]
what happened to example.js!

**Comment by 10maurycy10** [comment]
redaction!

**Comment by 10maurycy10** [comment]
fixed now.

**Pull request merged**